### PR TITLE
Update README source [skip CI]

### PR DESCRIPTION
**Pull request by ylavoie**
Fix typo in README source

### CODE REVIEW

File: doc/sources/_README.md [hunk 1]
```diff
@@ -86,9 +86,9 @@ because it lacks secure connections to protect your users' passwords.
 
 The system is installed and should be available for evaluation through
 
-* `http://localhost:5762/setup.pl`  
+* `http://localhost:5762/setup.pl`
```
Comment (ehuelsmann) [inline doc/sources/_README.md:89]: These trailing spaces were explicit, to force continuation of the content on the next line.

File: doc/sources/_README.md [hunk 2]
```diff
@@ -86,9 +86,9 @@ because it lacks secure connections to protect your users' passwords.
...
   Creation and privileged management of company databases
-* `http://localhost:5762/login.pl`  
+* `http://localhost:5762/login.pl`
```
Comment (ehuelsmann) [inline doc/sources/_README.md:91]: Same here.

File: doc/sources/_README.md (hunk 1 above)
Comment (ylavoie) [reply doc/sources/_README.md:89]: Fixed in #6379

File: doc/sources/_README.md (hunk 2 above)
Comment (ylavoie) [reply doc/sources/_README.md:91]: Fixed in #6379

**Pull request merged**